sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Release History
 
-## 4.0.0-beta.1 (Unreleased)
+## 4.0.0-beta.1 (2021-11-09)
 
 ### New Features
 
-- Added `CertificateClient` with  `GetCertificate()` and `GetCertificateVersion()` APIs.
+Initial preview of Azure Security Key Vault Certificates API for CPP.
+
+- Added `Azure::Security::KeyVault::Certificates::CertificateClient` for get, create, list, delete, backup, restore, and import certificate operations.
+- Added high-level and simplified `keyvault_certificates.hpp` file for simpler include experience for customers.
+- Added model types which are returned from the `CertificateClient` operations, such as `Azure::Security::KeyVault::Certificates::KeyVaultCertificate`.
```

Comment:
Nit: We typically put everything under top-level bullets in changelog/markdown (with sub-bullets), and avoid the term `preview` in favor of `beta`.

```suggestion
- Initial beta release of Azure Security Key Vault Certificates API for CPP.
  - Added `Azure::Security::KeyVault::Certificates::CertificateClient` for get, create, list, delete, backup, restore, and import certificate operations.
  - Added high-level and simplified `keyvault_certificates.hpp` file for simpler include experience for customers.
  - Added model types which are returned from the `CertificateClient` operations, such as `Azure::Security::KeyVault::Certificates::KeyVaultCertificate`.
```